client/cypress/integration/query/parameter_spec.js

```diff
@@ -247,6 +253,8 @@ describe("Parameter", () => {
           cy.wrap($option).click();
         });
 
+        openAndSearchAntdDropdown("value1"); // asserts option filter prop
```

Comment:
I initially thought the Query Based Parameters already had a similar test for only 1 value selection 😅. But I guess we can just create a new one (based on the Enum parameter "updates the results after selecting a value")?